android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
         versionCode 1
         versionName "1.0"
         multiDexEnabled true
+        vectorDrawables.useSupportLibrary = true
```

Comment:
what's this?

Reply:
It is required by react-native-image-crop-picker in order to work properly